### PR TITLE
chore: Bump cypari2 to 2.1.5 and document new Python requirements

### PR DESCRIPTION
**Pull request by gwhitney**
This will reconcile development and production versions, while maintaining the ability to develop on Ubuntu 20.04.

### CODE REVIEW

**Comment by gwhitney** [comment]
Kate, your situation is most relevant to this PR, hence the request to you if you have time before you go.

File: doc/install-ubuntu.md [hunk 1]
```diff
@@ -112,41 +115,67 @@ sudo apt install build-essential autoconf
...
+  above:
+
+  ```shell
+  sudo add-apt-repository universe
+  sudo apt update
+  ```
+
+  If/when this seems to have succeeded, execute `python3.9 --version`. It
+  should succeed and display a version number like 3.9.13 (in any case,
+  starting with 3.9.
```
Comment (katestange) [inline doc/install-ubuntu.md:167]: closing parenthesis?

**Review by katestange — CHANGES_REQUESTED** [review]
Works on my machine as advertised!

File: doc/install-ubuntu.md (hunk 1 above)
Comment (gwhitney) [reply doc/install-ubuntu.md]: Oops, thanks, should be better now.

**Review by katestange — APPROVED** [review]
Looks good now.

**Pull request merged**